README.md

```diff
@@ -1 +1,52 @@
-# TodoList
\ No newline at end of file
+# ToDo List App
+
+Todo List is a web application that allows users to add/remove the list of "to-dos". It provides an interface for adding to do activities, for visualizing the list of added tasks to complete. This is achieved using plain HTML, CSS, JavaScript objects, arrays, DOM, and basic events.
+
+
+![Screenshot](src/project_creenshot.png)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Git flow
+- GitHub
+
+### Prerequisites
+
+- Terminal (Bash)
+- Git
+- JavaScript
+
+### Setup: How to start the TicTacToe game
+
+- clone our repo: `https://github.com/Chike1990/ToDo_List`
+- cd into: `cd index.html`
+- run `index.html`
```

Comment:
Let's try not to mislead the newbies.

I think if you follow the steps you listed you won't be able to run the app.

Obviously, you can't cd a file, we can only cd a folder.

**Note:** You're supposed to set up [Webpack](https://webpack.js.org/guides/getting-started/#using-a-configuration) according to the project requirement.
And make sure CSS is loaded by Webpack Style/CSS Loader.

Then after successfully set up, you can now write the following steps.

```
- git clone your repo
- cd folder
- npm install
- npm start
```